src/Microsoft.AspNetCore.Mvc.Core/Internal/ResourceInvoker.cs

```diff
@@ -89,6 +89,12 @@ public abstract class ResourceInvoker
                 {
                     _logger.ExecutingAction(_actionContext.ActionDescriptor);
 
+                    _logger.ExecutingAuthorizationFilters(_filters);
+                    _logger.ExecutingResourceFilters(_filters);
```

Comment:
What if a filter execution fails at some point. Then some of these logs will become irrelevant...
Also, the actual order the filters are executed in can change, and that can result in these logs being out of order.
What about logging right where the filters are being called/executed ? So if for whatever reason a filter execution would fail, the rest of the messages for the filters which haven't been executed wouldn't even be logged.